src/app/modules/startup/module.nim

```diff
@@ -320,8 +320,17 @@ method emitObtainingPasswordSuccess*[T](self: Module[T], password: string) =
   self.view.emitObtainingPasswordSuccess(password)
 
 method finishAppLoading*[T](self: Module[T]) =
-  self.delegate.finishAppLoading()
   self.delegate.appReady()
+  let currStateObj = self.view.currentStartupStateObj()
+  if not currStateObj.isNil:
+    var eventType = "user-logged in"
+    if currStateObj.flowType() != FlowType.AppLogin:
+      eventType = "onboarding-completed"
+    singletonInstance.globalEvents.addCentralizedMetricIfEnabled(eventType,
+      $(%*{"flowType": currStateObj.flowType()}))
```

Comment:
Also... will become soon obsolete with the new onboarding. 

I noticed we already have access to it in QML, using the signal:
```qml
Global.addCentralizedMetricIfEnabled((string eventName, var eventValue)
```

wdyt?

Reply:
In the current code we have, it's way easier for me to put this call here because it's the last call before going to the main app.

If you have a similar spot in QML, I don't mind using it, but that would also change with the new onboarding no?

Reply:
it's fine for now :)